### PR TITLE
Adds a unique suicide to the hierophant club, plus some attack verbs

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
add: You can now kill yourself in fun and creative ways with the hierophant club.
/:cl:

A giant laser stick feels strangely lacking if you can't kill yourself in a neat way with it. Now you can.

### CODE REVIEW

**Comment by kevinz000** [comment]
if the blasts dont do damage they should go out way further tbh.

**Comment by Xhuis** [comment]
The 3x3 is consistent with the actual teleportation function, which the suicide imitates.

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -1028,6 +1029,23 @@
 	..()
 	to_chat(user, "<span class='hierophant_warning'>The[beacon ? " beacon is not currently":"re is a beacon"] attached.</span>")
 
+/obj/item/hierophant_club/suicide_act(mob/living/user)
+	user.say("Xverwpsgexmrk...")
+	user.visible_message("<span class='suicide'>[user] holds [src] into the air! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	new/obj/effect/temp_visual/hierophant/telegraph(get_turf(user))
+	playsound(user,'sound/machines/airlockopen.ogg', 75, TRUE)
+	sleep(3)
```
Comment (Cyberboss) [inline code/modules/mining/lavaland/necropolis_chests.dm]: if(QDELETED(user))
    return

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 2]
```diff
@@ -1028,6 +1029,23 @@
...
+	user.visible_message("<span class='hierophant_warning'>[user] fades out, leaving their belongings behind!</span>")
+	for(var/obj/item/I in user)
+		if(I != src)
+			user.dropItemToGround(I)
+	for(var/turf/T in RANGE_TURFS(1, user))
+		var/obj/effect/temp_visual/hierophant/blast/B = new(T, user, TRUE)
+		B.damage = 0
+	user.dropItemToGround(src) //Drop us last, so it goes on top of their stuff
+	qdel(user)
+	return
```
Comment (Cyberboss) [inline code/modules/mining/lavaland/necropolis_chests.dm]: remove trailing return

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Above

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 3]
```diff
@@ -1028,6 +1029,24 @@
 	..()
 	to_chat(user, "<span class='hierophant_warning'>The[beacon ? " beacon is not currently":"re is a beacon"] attached.</span>")
 
+/obj/item/hierophant_club/suicide_act(mob/living/user)
+	user.say("Xverwpsgexmrk...")
```
Comment (ChangelingRain) [inline code/modules/mining/lavaland/necropolis_chests.dm]: I think you should not have any lines in the hierophant's rotated text that aren't from it.
like, really, it's a big machine dealy, at least have the club itself speak instead.

**Review by Cyberboss — APPROVED** [review]

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 4]
```diff
@@ -1028,6 +1029,24 @@
 	..()
 	to_chat(user, "<span class='hierophant_warning'>The[beacon ? " beacon is not currently":"re is a beacon"] attached.</span>")
 
+/obj/item/hierophant_club/suicide_act(mob/living/user)
+	say("Xverwpsgexmrk...")
+	user.visible_message("<span class='suicide'>[user] holds [src] into the air! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	new/obj/effect/temp_visual/hierophant/telegraph(get_turf(user))
+	playsound(user,'sound/machines/airlockopen.ogg', 75, TRUE)
+	sleep(3)
```
Comment (Cyberboss) [inline code/modules/mining/lavaland/necropolis_chests.dm]: Actually, timer instead please

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**